site/src/comparison.rs

```diff
@@ -1040,6 +1040,15 @@ impl TestResultComparison {
         Some(change.abs() / threshold)
     }
 
+    /// Whether the comparison is relevant or not
+    fn is_relevant(&self) -> bool {
```

Comment:
This naming here is still a bit weird w.r.t `RelevanceLevel`. First "non-relevant" comparisons are filtered out and then the relevance level of the whole comparison set is computed, and it assumes that "non-relevant" comparisons were already filtered out. It's a bit confusing that relevance here is a bool that concerns a single comparison, while a bit later relevance is an enum that concerns the whole comparison.

I wonder if this should have a different name or if `is_relevant` and `relevance_level` should be somehow merged together. But it's not a big deal IMO.

Reply:
Relevance accounts for both significance and magniture, right? So it's confusing that this function only looks at magnitude.

Reply:
So, this is a bit of confusing point. Magnitude is partially based on significance since one of the two parts of magnitude is the magnitude above the significance threshold. So it's very unlikely that a non-significant change will have a large enough magnitude to be considered consistent. 

Relevance has not been defined directly by being significant only by having a large enough magnitude. 
However, this seems to be causing confusion so I think we can just close the gap and say that relevance is a function of both significance and magnitude.